feature-group-definitions/constructed-stylesheets.dist.yml

```diff
@@ -0,0 +1,22 @@
+# Generated from: constructed-stylesheets.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Constructed stylesheets
+description: The `CSSStyleSheet` constructor creates a new stylesheet which can be shared between a document and multiple shadow roots using `adoptedStyleSheets`.
+spec:
+  - https://drafts.csswg.org/cssom-1/#dom-cssstylesheet-cssstylesheet
+  - https://drafts.csswg.org/cssom-1/#dom-documentorshadowroot-adoptedstylesheets
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/2846
+status:
+  baseline: false
```

Comment:
This is probably caused by the note in https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/CSSStyleSheet#browser_compatibility

Need to review and do an editorial override.